src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+@include respond-to(large) {
+  .CollectionList-wrapper {
```

Comment:
If I remember correctly, we tend to put the mixins inside the class definitions, not the other way around.